blog/_posts/2018-11-18-TeamSectumsempraGHCI.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Team Sectumsempra at GHCI 2018!
+layout: post
+created_at: Tue Dec 18 2018
+permalink: blog/2018-11-18-TeamSectumsempraGHCI
```

Comment:
as in the last PR, please change the permalink and filename date here to be `2018-12-18`.